src/test/java/teammates/test/cases/browsertests/InstructorHomePageUiTest.java

```diff
@@ -226,82 +226,56 @@ private void testRemindActions() {
-        homePage.clickAndCancel(homePage.getRemindInnerLink(feedbackSessionOpen.getCourseId(),
-                                                            feedbackSessionOpen.getFeedbackSessionName()));
-        homePage.clickRemindOptionsLink(feedbackSessionOpen.getCourseId(), feedbackSessionOpen.getFeedbackSessionName());
-        homePage.clickAndConfirm(homePage.getRemindInnerLink(feedbackSessionOpen.getCourseId(),
-                                                             feedbackSessionOpen.getFeedbackSessionName()));
-        homePage.waitForPageToLoad();
-        homePage.verifyStatus(Const.StatusMessages.FEEDBACK_SESSION_REMINDERSSENT);
 
         ______TS("remind particular users action: OPEN feedback session");
 
-        homePage.clickRemindOptionsLink(feedbackSessionOpen.getCourseId(), feedbackSessionOpen.getFeedbackSessionName());
         homePage.clickRemindParticularUsersLink(feedbackSessionOpen.getCourseId(),
                                                 feedbackSessionOpen.getFeedbackSessionName());
         homePage.cancelRemindParticularUsersForm();
 
-        homePage.clickRemindOptionsLink(feedbackSessionOpen.getCourseId(), feedbackSessionOpen.getFeedbackSessionName());
         homePage.clickRemindParticularUsersLink(feedbackSessionOpen.getCourseId(),
                                                 feedbackSessionOpen.getFeedbackSessionName());
         homePage.waitForAjaxLoaderGifToDisappear();
         homePage.submitRemindParticularUsersForm();
         homePage.waitForPageToLoad();
         homePage.verifyStatus(Const.StatusMessages.FEEDBACK_SESSION_REMINDERSEMPTYRECIPIENT);
-        homePage.clickRemindOptionsLink(feedbackSessionOpen.getCourseId(), feedbackSessionOpen.getFeedbackSessionName());
         homePage.clickRemindParticularUsersLink(feedbackSessionOpen.getCourseId(),
                                                 feedbackSessionOpen.getFeedbackSessionName());
         homePage.waitForAjaxLoaderGifToDisappear();
         homePage.fillRemindParticularUsersForm();
         homePage.submitRemindParticularUsersForm();
         homePage.waitForPageToLoad();
         homePage.verifyStatus(Const.StatusMessages.FEEDBACK_SESSION_REMINDERSSENT);
-
-        ______TS("remind action: CLOSED feedback session - inner button");
-
-        homePage.clickRemindOptionsLink(feedbackSessionClosed.getCourseId(), feedbackSessionClosed.getFeedbackSessionName());
-        homePage.clickAndCancel(homePage.getRemindInnerLink(feedbackSessionClosed.getCourseId(),
-                feedbackSessionOpen.getFeedbackSessionName()));
-        homePage.clickRemindOptionsLink(feedbackSessionClosed.getCourseId(), feedbackSessionClosed.getFeedbackSessionName());
-        homePage.clickAndConfirm(homePage.getRemindInnerLink(feedbackSessionClosed.getCourseId(),
-                feedbackSessionClosed.getFeedbackSessionName()));
+        homePage.clickRemindParticularUsersLink(feedbackSessionOpen.getCourseId(),
```

Comment:
Mind explaining what the test cases here are testing now, as compared to before?

Reply:
Before, the test cases tested that the outer remind button (remind all not submitted) and the inner buttons (remind all, remind particular) can be clicked and the remind particular modal behaves as expected (cancel, submit). Now, there is no more outer button or dropdown buttons, the remind button now opens the remind modal, so the new tests test the button's and the modal's behaviours, and test the 2 new checkboxes inside the modal.